### DISCUSSION TITLE
Random seed for ocr_predictor

### DISCUSSION THREAD

**Discussion by hienphan161** [mindee/doctr#1753] [Q&A]
Is there any way to set a random seed for the `ocr_predictor`? 

Currently, when calling `ocr_predictor` multiple times with one image, it returns different outputs sometimes. 

I tried setting `torch.manual_seed(0)` in the script where the `ocr_predictor` is called but it didn't work.

**Comment by felixdittrich92** [comment]
This should normally not be the case do you have an example to reproduce ?

If you want to seed you should also seed numpy and random :)
